examples/c/test_tracker.c

```diff
@@ -44,7 +44,11 @@
 int main(int arg, char** args) {
     int i;
     int count = psmove_count_connected();
+#ifdef _MSC_VER
+    PSMove* controllers[8];
```

Comment:
That's really bad. What if more than 8 controllers are connected? We have this for example in Edgar Rice Soiree (20 controllers connected). Instead, if MSVC is unable to make a variable-sized local array, use `calloc` and make sure you later `free` it before exiting the function:

```
int count = psmove_count_connected();
PSMove **controllers = (PSMove **)calloc(count, sizeof(PSMove *));

// ...

free(controllers);
```

If you move the `calloc` below the block that says:

```
printf("### Found %d controllers.\n", count);
if (count == 0) {
    return 1;
}
```

then you just need a single `free` call between `psmove_tracker_free(tracker);` and `return 0;` at the end of the function.